COMMUNICATON.md

```diff
@@ -0,0 +1,33 @@
+# Communication
+
+## Code of conduct
+
+This project has adopted the [GitHub community guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines).
+
+## Issues
+
+Issues are used as the primary method for tracking anything to do with the Fairlearn project.
+
+See the [README](https://github.com/fairlearn/fairlearn/blob/master/CONTRIBUTING.md#issues)
+for information on how to file an issue if you think you have found a bug or have a feature request.
+
+## Proposals
```

Comment:
maybe call it `API Design`?

Reply:
Agreed we should have more clarity about what we mean by "proposals". I like the idea of starting with `API Design`. As I say above, I think that this part belongs into `CONTRIBUTING.md`.

Eventually, I'd like to also include proposals for content standards. The key one is around writing Jupyter notebooks, to make sure that they do not actually create more harm, e.g., by creating an illusion of "solving fairness".